src/geocat/f2py/dpres_plevel_wrapper.py

```diff
@@ -153,10 +158,17 @@ def _dpres_plevel(plev, psfc, ptop, msg_py):
     return dp
 
 
-# TODO: WIll revisit this after merging sanity check module to repo.
-def _sanity_check(pressure_levels, pressure_surface, pressure_top):
-    # ''' Basic sanity checks
+# Helper function that checks the validity of the input
+
+
+def _validity_check(pressure_levels, pressure_surface, pressure_top):
+    # ''' Basic validity checks
+    is_input_xr = True
+
     if not isinstance(pressure_levels, xr.DataArray):
+
+        is_input_xr = False
+
```

Comment:
if _validity_check will be used, perhaps using check from checks.py for this functions internals would be a good idea. But it does not need to happen as part of this pull request.

Reply:
I noticed we had never used `check()` from checks.py after your comment. Maybe let us get this merged as is but use `check()` instead of several checks in each function in a separate PR?

Reply:
Sure, I'm not a fan of mutiple functions with the same `_validity_check` name, but if it's not breaking things, and you plan to refactor it later, then I'm ok with leaving it as-is.